pallets/vector/src/tests.rs

```diff
@@ -182,7 +179,7 @@ fn test_fulfill_step_call_proof_not_valid() {
 
 		let result = Bridge::fulfill_call(
 			RuntimeOrigin::signed(TEST_SENDER_ACCOUNT),
-			STEP_FN_ID,
+			STEP_FUNCTION_ID,
```

Comment:
In general, we should use the step function id that we have stored inside our storage. There is no guarantee that `STEP_FUNCTION_ID` is equal to the value that we have stored.

In any case, no changes are needed but for future tests let's do it properly.